queries.sql

```diff
@@ -52,3 +52,11 @@ select neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 select species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 select species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' GROUP BY species;
 
+SELECT animals.name, owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody';
+SELECT animals.name AS animal_name, species.name AS species_type FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
+SELECT species.name, count(animals) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell';
+SELECT species.name, count(animals) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
```

Comment:
- [ ] Please List all Digimon owned by Jennifer Orwell, I see that you specify that the owner is Jennifer Orwell but you didn't add the condition where the animal is Digimon.

Reply:
Done